app/mailers/user_mailer.rb

```diff
@@ -436,14 +436,24 @@ def account_reset_deletion_period_interval
 
     distance_of_time_in_words(
       current_time,
-      current_time + IdentityConfig.store.account_reset_wait_period_days.days,
+      current_time + account_reset_wait_period_days,
       true,
-      accumulate_on: :hours,
+      accumulate_on: reset_accumulation_type,
     )
   end
 
-  def account_reset_deletion_period_hours
-    IdentityConfig.store.account_reset_wait_period_days.days.in_hours.to_i
+  def account_reset_wait_period_days
```

Comment:
Would it be possible to use `AccountResetConcern` here or find another way to share common code?